Sources/HAP/Base/Generated.swift

```diff
@@ -44,6 +44,47 @@ public enum AccessoryType: String, Codable {
 	case audioReceiver = "34"
 	case televisionSetTopBox = "35"
 	case televisionStreamingStick = "36"
+
+    public var description: String {
```

Comment:
I prefer to have a switch-statement here instead.

Reply:
OK given this type is restricted to only specific cases I have made the description use a switch statement

Reply:
I still think it should use a switch, like my suggested code. Instead of a fatalError you could return the value in the default case.

Reply:
While I much prefer the neatness of the dictionary lookup, I have made the change you request for both ServiceType and CharacteristicType.